src/jsonpatcherproxy.js

```diff
@@ -143,7 +144,9 @@ var JSONPatcherProxy = (function() {
         return Reflect.deleteProperty(target, key);
       }
     });
-    return proxy;
+    /* keeping track of all the proxies to be able to revoke them later */
+    this.proxifiedObjectsMap.push(proxy);
+    return proxy.proxy;
   };
   //grab tree's leaves one by one, encapsulate them into a proxy and return
   JSONPatcherProxy.prototype._proxifyObjectTreeRecursively = function(
```

Comment:
@warpech I started with this yesterday but figured that would render the child object useless even if it was cached earlier on purpose for later usages, [like in this demo](https://jsbin.com/barexugoca/1/edit?js,console). Doing it in Palindrom made more sense since `Palindrom.obj` is ours and we give it to the user for use. But the observed object in JSONPatcherProxies is not ours and I guess we should leave it, and its children, usable. 

What do you think?